packages/admin/src/Charcoal/Admin/Service/SelectizeRenderer.php

```diff
@@ -123,6 +123,6 @@ public function renderTemplate($templateIdent, $context, $controllerIdent = null
             $template = $context;
         }
 
-        return $this->view->render($templateIdent, $template);
+        return $this->view->renderTemplate($templateIdent, $template);
```

Comment:
The method is named `renderTemplate` but uses `$templateIdent` as a parameter name.
I think the Selectize templating (`item_render` and `option_render`) could possibly be either a template file path or a template string. For example, a complex media card component for each option might be easier to manage in its own template file instead of inline within some object's metadata.

I suppose if templating must be in its own file, one can use `{{> path/to/my/selectize/item }}`.

Reply:
@JoelAlphonso What do you think about Selectize templating? Should we continue to support view file paths and templating strings or just the latter?

We could add a condition in this method to check if its a template string or if there's a template file matching the potential path, like is done in `AbstractLoader::load()`.

Reply:
Maybe something like that?

```
        if ($this->view->isTemplateString($templateIdent)) {
            return $this->view->render($templateIdent, $template);
        }

        return $this->view->renderTemplate($templateIdent, $template);
```